bblfsh-web/templates/deployment.yaml

```diff
@@ -34,3 +34,19 @@ spec:
       nodeSelector:
 {{ toYaml . | indent 8 }}
     {{- end }}
+
+    {{- with .Values.affinity }}
+      affinity:
+{{ toYaml . | indent 8 }}
+    {{- end}}
+{{- if .Values.antiAffinityEnabled }}
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+             matchExpressions:
+              - key: app
+                operator: In
+                values:
+                - {{ template "fullname" . }}
+            topologyKey: kubernetes.io/hostname
+{{- end }}
```

Comment:
You're mixing two different paradigms here: one to set `affinity` freely from the values and the other to set a certain affinity mechanism in the chart and both can compete if we add enough keys.

I'd go for the more open one here
```
    {{- with .Values.affinity }}
      affinity:
{{ toYaml . | indent 8 }}
    {{- end}}
```
and then leave the users set the affinity the way they want. In our case, we will set this in the values as we want these charts in this repo to be as open as possible.

Reply:
Changed to use the affinity block as requested. 
No more affinity enabled by default.